minikube-gradle-plugin/src/main/java/com/google/cloud/tools/minikube/MinikubePlugin.java

```diff
@@ -49,23 +51,28 @@ private void configureMinikubeTaskAdditionCallback() {
             });
```

Comment:
I think you can add commandExecutorFactory in here, so it's applied to our and custom user tasks?

Reply:
Done